src/components/statusBar.jsx

```diff
@@ -1,11 +1,38 @@
 import React, { Component } from 'react';
+import '../styles/css/statusBar.css';
+
+export default class StatusBar extends Component {
+  _getStatusClass = player => {
+    debugger;
+    let classes = 'col';
+    classes +=
+      this.props.currentTurn === player ? ' currentTurn' : ' inactivePlayer';
+    return classes;
+  };
+
+  _getResult = () => {
+    const { winner } = this.props;
+    if (winner === '') {
```

Comment:
Refactor this to use JSON object instead of IF-ELSE.

Reply:
Resolved by using ternary operation instead of if{} else{} block